showcase/scripts/showcase_utilities.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+function generateShowcaseIfMissing {
```

Comment:
This function is only used once, I don't think we need another file for it.

Reply:
I removed this file and put the logic in the script that uses it